x/btcstaking/types/btcstaking.go

```diff
@@ -107,3 +108,12 @@ func GetOrderedCovenantSignatures(fpIdx int, covSigsList []*CovenantAdaptorSigna
 
 	return orderedCovSigs, nil
 }
+
+// NewLargestBtcReOrg creates a new Largest BTC reorg based on the rollback vars
+func NewLargestBtcReOrg(rollbackFrom, rollbackTo *btclightclienttypes.BTCHeaderInfo) LargestBtcReOrg {
+	return LargestBtcReOrg{
+		BlockDiff:    rollbackFrom.Height - rollbackTo.Height,
```

Comment:
Do we need to ensure that `rollbackFrom.Height > rollbackTo.Height`?

Reply:
we do check this in `CheckRollBackInvariants`